midend/removeParameters.cpp

```diff
@@ -148,18 +148,21 @@ const IR::Node* DoRemoveActionParameters::postorder(IR::MethodCallExpression* ex
     return expression;
 }
 
-//////////////////////////////////////////
-
+/**
+ * This is needed because of this case:
+ *
+ * \code{.cpp}
```

Comment:
This comment is in the wrong place, it was explaining why MoveDeclarations is required. This is not what this pass is doing. The comment for the pass is in the header file. This is a code comment.